base/shell/explorer/traywnd.cpp

```diff
@@ -264,6 +264,95 @@ class CStartButton
 
 };
 
+static const WCHAR szTrayShowDesktopButton[] = L"TrayShowDesktopButtonWClass";
+
+// The 'Show Desktop' button at edge of taskbar
+class CTrayShowDesktopButton :
+    public CComCoClass<CTrayShowDesktopButton>,
+    public CComObjectRootEx<CComMultiThreadModelNoCS>,
+    public CWindowImpl<CTrayShowDesktopButton, CWindow, CControlWinTraits>
+{
+public:
+    DECLARE_WND_CLASS_EX(szTrayShowDesktopButton, CS_DBLCLKS | CS_HREDRAW | CS_VREDRAW,
```

Comment:
Do you need `CS_DBLCLKS` ?

Reply:
The user might want to double-click my head.

Reply:
Did you test that this works?

Reply:
Quickly repeating clicking is working well.

Reply:
And what would happen *exacly*, if you would not have that CD_DBL_CLKS? Can you try to describe the samptom with words?

Reply:
`CS_DBLCLKS` and handler are removed.